operations/hpe_pdu/hpe_pdu_admin_procedures.md

```diff
@@ -36,9 +59,11 @@ The following is needed before running this procedure:
 
    Enter the root password for `ncn-m001` when prompted.
 
-2. Connect to [`https://localhost:8443`](https://localhost:8443) using a web browser.
+1. Connect to [`https://localhost:8443`](https://localhost:8443) using a web browser.
+
+1. Log in with the `admin` username. Enter the admin password. If the admin password has not been changed, there will be a prompt to change the password.
 
-3. Log in with the `admin` username. Enter the admin password. If the admin password has not been changed, there will be a prompt to change the password.
+1. If the `admin` password was changed, update Vault with the new credentials following procedure [Update Vault Credentials](#update-vault-credentials)
```

Comment:
I think this line should be removed, as the section it is referencing no longer exists.
```
1. If the `admin` password was changed, update Vault with the new credentials following procedure [Update Vault Credentials](#update-vault-credentials)
```

Reply:
removed